src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  test('should return true if an isogram (playgrounds)', () => {
+    expect(isIsogram('playgrounds')).toBe(true);
```

Comment:
try to use .toBeTruthy() or .toBeFalsy()

Reply:
fixed